GoogleDataTransport/GDTCORTests/Unit/GDTCORDatabaseTest.m

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "GDTCORTests/Unit/GDTCORTestCase.h"
+
+#import "GDTCORLibrary/Private/GDTCORDatabase_Private.h"
+#import "GDTCORLibrary/Public/GDTCORDatabase.h"
+
+@interface GDTCORDatabaseTest : GDTCORTestCase
+
+/** */
+@property(nonatomic) GDTCORDatabase *db;
+
+@end
+
+@implementation GDTCORDatabaseTest
+
+- (void)setUp {
+  [super setUp];
+  NSDictionary *migrations = @{@1 : @"CREATE TABLE \"GDTCORDatabaseTest\" (\"some_text\" TEXT);"};
+  _db = [[GDTCORDatabase alloc] initWithURL:nil migrationStatements:migrations];
+  XCTAssertNotNil(_db);
+}
+
+- (void)tearDown {
+  [super tearDown];
+  dispatch_sync(_db.dbQueue, ^{
+                });
+  [_db close];
+  _db = nil;
+}
+
+/** */
+- (void)testInstantiation {
+  NSString *dbPath = [NSTemporaryDirectory() stringByAppendingPathComponent:@"test.sqlite3"];
```

Comment:
Do you want to randomize the db name for concurrently running tests?

Reply:
Most tests create in-memory stores, but these tests aren't going to be run concurrently anyway.